circle.yml

```diff
@@ -0,0 +1,69 @@
+    - rm -rf ~/.pyenv;
+    - rm -rf ~/virtualenvs;
+    # Get Anaconda and conda-based requirements
+    - >
+      if [ ! -d "/home/ubuntu/miniconda" ]; then
+        echo "Setting up conda";
+        wget -q http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O ~/miniconda.sh;
+        chmod +x ~/miniconda.sh;
+        ~/miniconda.sh -b -p /home/ubuntu/miniconda;
+        conda update --yes --quiet conda;
+        conda create -n circleenv --yes pip python=2.7 pip numpy scipy scikit-learn mayavi matplotlib sphinx==1.2.3 pillow six IPython;
+        sed -i "s/ENABLE_USER_SITE = .*/ENABLE_USER_SITE = False/g" /home/ubuntu/miniconda/envs/circleenv/lib/python2.7/site.py;
+      else
+        echo "Conda already set up.";
+      fi;
+    - ls -al /home/ubuntu/miniconda;
+    - ls -al /home/ubuntu/miniconda/bin;
+    - echo $PATH;
+    - which python;
+    - which pip;
+    - git clone https://github.com/sphinx-gallery/sphinx-gallery.git;
+    - cd sphinx-gallery && pip install -r requirements.txt && python setup.py develop;
+    - cd /home/ubuntu && git clone https://github.com/enthought/pyface.git && cd pyface && python setup.py develop;
+    - pip install sphinx_bootstrap_theme PySurfer nilearn neo;
+    - cd /home/ubuntu/mne-python && python setup.py develop;
+    - mkdir -p ~/mne_data;
+    - python -c "import mne; mne.datasets.sample.data_path(verbose=True);";
+    - python -c "import mne; mne.datasets.testing.data_path(verbose=True);";
+    - python -c "import mne; mne.datasets.spm_face.data_path(verbose=True);";
+    - python -c "import mne; mne.datasets.somato.data_path(verbose=True);";
+    - python -c "import mne; mne.datasets.brainstorm.bst_auditory.data_path(verbose=True)" --accept-brainstorm-license;
+    - python -c "import mne; mne.datasets.brainstorm.bst_resting.data_path(verbose=True);" --accept-brainstorm-license;
+    - python -c "import mne; mne.datasets.brainstorm.bst_raw.data_path(verbose=True);" --accept-brainstorm-license;
+    - python -c "import mne; mne.sys_info()";
+    - python -c "import mne, os; print(mne.datasets.sample.data_path(os.path.join(os.environ['HOME'], 'mne_data'), verbose=True))";
+    - /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -screen 0 1400x900x24 -ac +extension GLX +render -noreset;
+
+test:
+  override:
+    - cd
+
+deployment:
+  staging:
+    branch: master
+    commands:
+      - case $CIRCLE_NODE_INDEX in 0) cd doc && make html_dev-pattern PATTERN='/tutorials/plot_' ;; 1) cd doc && make html_dev-pattern PATTERN='/examples/plot_\|/connectivity/plot_\|/datasets/plot_\|/decoding/plot_\|/forward/plot_\|/inverse/plot_\|/io/plot_\|/preprocessing/plot_\|/realtime/plot_\|/simulation/plot_\|/stats/plot_\|/time_frequency/plot_\|/visualization/plot_' ;; esac:
+          parallel: true
+          timeout: 1500
+
+general:
+  # Open the doc to the API
+  artifacts:
+    - "doc/_build/html"
```

Comment:
There is a `cd doc` in your `commands` so I wonder if you aren't in the right directory by this point to use a relative path. Maybe make it absolute to be safe?